task-list/app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1><%= @person.name %>'s Tasks</h1>
+
+  <div class = "tasks">
+  <table>
+    <tbody>
+      <% @person.tasks.each do |task| %>
+        <%if task.completed_at == nil %>
```

Comment:
It is preferred to use the `.nil?` method as opposed the checking for equivalence to `nil`